package.json

```diff
@@ -112,7 +113,7 @@
   "bundlesize": [
     {
       "path": "./dist/DO-NOT-USE.min.js",
-      "threshold": "3 Kb"
+      "threshold": "6 Kb"
```

Comment:
Well, that's acceptable to me. Probably need a seperate preact bundle again since preact don't need to filter props.